packages/core/src/docs/colors.md

```diff
@@ -54,7 +54,7 @@ Blueprint provides variables for colors in Sass, Less, and JavaScript.
 Example in Sass:
 
 ```scss
-@import "~@blueprintjs/core/lib/scss/variables";
+@import "@blueprintjs/core/lib/scss/variables";
```

Comment:
should we be recommending `@use "@blueprintjs/core/lib/scss/variables" as bp` instead?

Reply:
I will change the recommendation once we switch to `@use` in this repo